persistence/scaladsl/src/main/scala/com/lightbend/lagom/internal/scaladsl/persistence/PersistentEntityActor.scala

```diff
@@ -39,6 +34,24 @@ private[lagom] object PersistentEntityActor {
    * with persistent actors.
    */
   case object Stop
+
+  val EntityIdSeparator = '-'
+
+  /**
+   * @return persistence id split into (persistenceIdPrefix, entityId)
+   */
+  def splitPersistenceId(persistenceId: String): (String, String) = {
```

Comment:
object is `private[lagom]` so method is not public.

I'll change it into `extractEntityId(String): String`